gamemodes/terrortown/gamemode/client/cl_search.lua

```diff
@@ -673,12 +673,14 @@ local function TTT2ConfirmMsg()
 		tbl.team = LANG.GetTranslation(net.ReadString())
 	end
 
-	local img = Material("vgui/ttt/icon_corpse")
-
-	if sid64 ~= "" then
-		img = draw.GetAvatarMaterial(sid64, "medium", img)
+	-- checking for bots
+	if sid64 == "" then
+		sid64 = nil
 	end
 
+	local img = Material("vgui/ttt/icon_corpse")
+	img = draw.GetAvatarMaterial(sid64, "medium", img)
```

Comment:
I think fallbacks should be handled in place rather than invisible inside the function, we might want to think about changing that.